bigtable/bttest/inmem.go

```diff
@@ -1334,7 +1362,15 @@ func (r *row) gc(rules map[string]*btapb.GcRule) {
 			continue
 		}
 		for col, cs := range fam.cells {
-			r.families[fam.name].cells[col] = applyGC(cs, rule)
+			cs = applyGC(cs, rule)
+			if len(cs) == 0 {
+				delete(fam.cells, col)
```

Comment:
above there is a note about iterating & deleting being a possible issue for rows. Why is it ok for columns and families?

Reply:
It's OK to manipulate the row itself as we're holding a lock on the row (grabbed further up the call stack).

Reply:
Why not move the lock/unlock calls to this method? (I know it makes the diff bigger, but it makes the code easier to follow.)